grasa_event_locator/forms.py

```diff
@@ -28,7 +28,7 @@
     ("Tutoring", "Tutoring"),
     ("Other", "Other"),
 ]
-transportationList = [("Provided", "Provided"), ("Not-Provided", "Not-Provided")]
+transportationList = [("Provided", "Provided")]
```

Comment:
This change was made because the filters are now opt-in, i.e. you can choose to filter by events that provide transportation by checking it, or you can not filter. Is this correct?